### PR TITLE
feat: Describe unmanaged tenant permissions + import

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

File: docs/administration.index.md [hunk 1]
```diff
@@ -28,45 +28,47 @@ See the [meshWorkspace](meshcloud.workspace.md) documentation for details about
...
+| &nbsp;&nbsp;[Landing&nbsp;Zones](administration.landing-zones.md)                                                   |       &#10003;       |       &#10003;       |     &#10003;      |             |            |                    |                    |                      |
+| &nbsp;&nbsp;[Usage&nbsp;Reports](administration.usage.md)                                                           |       &#10003;       |       &#10003;       |                   |             |  &#10003;  |                    |                    |                      |
+| &nbsp;&nbsp;[Tenants](administration.tenants.md)                                                                    |       &#10003;       |       &#10003;       |     &#10003;      |  &#10003;   |            |                    |                    |         &#10003;     |
+| &nbsp;&nbsp;View [Unmanaged Tenants](administration.unmanaged-tenants.md)                                           |       &#10003;       |       &#10003;       |     &#10003;      |             |            |                    |                    |         &#10003;     |
```
Comment (Jelledb) [inline docs/administration.index.md:61]: Not sure why the diff is so messed up but I only added these two lines.

Also it's still TBD who can import unmanaged tenants

File: docs/administration.unmanaged-tenants.md [hunk 2]
```diff
@@ -20,13 +20,30 @@ Viewing unmanaged tenants can be easily done within the administration area. In
...
-
-> Adopting a tenant into meshStack can be done via the meshObject API.
+It is also recorded when the unmanaged tenant was last observed. If the unmanaged tenant has not been observed for a few days meshStack will
+assume that it has been deleted in the cloud platform and will remove it from the list.
 
 The screenshot below depicts how the unmanaged tenant list could look like.
 
 ![Unmanaged Tenants](assets/unmanaged-tenants.png)
+
+## Importing Unmanaged Tenants
```
Comment (uleMeshcloud) [inline docs/administration.unmanaged-tenants.md:30]: d: We have call it "Assign to Project"

File: docs/administration.unmanaged-tenants.md [hunk 3]
```diff
@@ -20,13 +20,30 @@ Viewing unmanaged tenants can be easily done within the administration area. In
...
+
+You can easily import an unmanaged tenant into meshStack by doing the following:
+
+- Click the Import button as depicted in the screenshot
+- In the prompt that opens up, select to which workspace & project this tenant should belong
+- Select the Landing Zone that should be applied to this tenant.
+  - The Landing Zone will automatically be applied after the first tenant replication and the
+    tenant will e.g. be assigned in the resource hierarchy in the cloud platform
+
+![Unmanaged Tenant Import](assets/unmanaged-tenants-import.png)
```
Comment (uleMeshcloud) [inline docs/administration.unmanaged-tenants.md:43]: d: new screenshot needed
Maybe we should stick to color blue when pointing on sth. I think this was mostly used in our screenshots

File: docs/administration.index.md (hunk 1 above)
Comment (uleMeshcloud) [reply docs/administration.index.md:61]: only Partner Admin can assign the tenant to project.
Other than that everything is fine, except maybe for the new term  we use

**Review by uleMeshcloud — APPROVED** [review]
lgtm only smaller points regarding the wording and the screenshot

**Pull request merged**